primitives/benches/pcs.rs

```diff
@@ -36,7 +36,7 @@ mod bench {
                 10
             };
 
-            let poly = Arc::new(DenseMultilinearExtension::rand(nv, &mut rng));
+            let poly = Rc::new(DenseMultilinearExtension::rand(nv, &mut rng));
```

Comment:
single threaded benchmarks only?

Reply:
changed back in [d1e8c3b](https://github.com/EspressoSystems/jellyfish/pull/254/commits/d1e8c3b1667052923932283cb68a3788c7252c42)